src/patternfly/components/Accordion/examples/Accordion.md

```diff
@@ -294,6 +295,62 @@ cssPrefix: pf-v5-c-accordion
 {{/accordion}}
 ```
 
+### Toggle icon at start
+
+```hbs
+{{#> accordion accordion--IsStartAligned="true"}}
```

Comment:
It's a new thing, but we can now use `true` and `false` as booleans instead of the string (which returns true for the hbs param regardless the value (eg `param="[true/false/chocolate chip cookie]"` all return true) 🥳

```suggestion
{{#> accordion accordion--IsStartAligned=true}}
```

Reply:
🤯 such a small thing but it feels like a game changer to me 😆